x-pack/legacy/plugins/siem/public/components/embeddables/embedded_map.tsx

```diff
@@ -152,11 +185,23 @@ export const EmbeddedMap = React.memo<EmbeddedMapProps>(
     }, [startDate, endDate]);
 
     return isError ? null : (
-      <>
+      <Embeddable>
+        <EmbeddableHeader title={i18n.EMBEDDABLE_HEADER_TITLE}>
+          <EuiText size="xs">
+            <EuiLink
+              href={`${ELASTIC_WEBSITE_URL}guide/en/siem/guide/${DOC_LINK_VERSION}/conf-map-ui.html`}
+              target="_blank"
+            >
+              {i18n.EMBEDDABLE_HEADER_HELP}
+            </EuiLink>
+          </EuiText>
+        </EmbeddableHeader>
+
         <InPortal node={portalNode}>
           <MapToolTip />
         </InPortal>
-        <EmbeddableWrapper>
+
+        <EmbeddableMap maintainRatio={!isIndexError}>
           {embeddable != null ? (
```

Comment:
With the new aspect ratio changes it seems theres be a bit of jitter when switching from rendering the loader to the error state:

<img width=300 src="https://user-images.githubusercontent.com/2946766/67591514-d66a1e80-f71a-11e9-966d-444669a5b738.png"/> <img width=300 src="https://user-images.githubusercontent.com/2946766/67591541-e3870d80-f71a-11e9-88d1-09fa1c107078.png"/>

Not sure there's too much we can do about this though, and I would prefer to optimize for the non-error state (which is what this does).

Reply:
Yeah, dropping the ratio when an error occurs seemed to be the best of our options. Otherwise, it would jump after map load, which wouldn't look so hot.